docs/data/routes/docs/nextjs/deploying-to-production/export/en.md

```diff
@@ -3,4 +3,76 @@ name: export
 routeTemplate: ./data/component-templates/article.yml
 title: Using `next export` with the Next.js sample app
 ---
-# Walkthrough: Using `next export` with the Next.js sample app
\ No newline at end of file
+# Walkthrough: Using `next export` with the Next.js sample app
+
+You can serve your JSS Next.js application as static HTML files using any static hosting service or content delivery network. 
+
+> Static HTML export does not support: 
+>
+> * Internationalized routing.
+> * Request rewrites.
+> * Visitor identification. 
+> * The default image `loader` used by the `next/image` component.
+>
+> See the  Next.js  [Static HTML Export Caveats](https://nextjs.org/docs/advanced-features/static-html-export/) documentation for more information.
+
+To export your JSS Next.js application as static HTML files, you must: 
+
+1. [Prepare your application for static HTML export]().
```

Comment:
Do we need header links here?

Reply:
Hmmm, they probably make little sense. Especially if someone jumps straight to export :D Will fix.